lib/node_modules/@stdlib/blas/base/sdot/lib/sdot.js

```diff
@@ -0,0 +1,97 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2019 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+var float64ToFloat32 = require( '@stdlib/number/float64/base/to-float32' );
+
+
+// VARIABLES //
+
+var M = 5;
+
+
+// MAIN //
+
+/**
+* Computes the dot product of `x` and `y`.
+*
+* @param {PositiveInteger} N Number of values to use from each array
```

Comment:
Missing hyphen separating parameter name from description and need to lowercase the first description word.